pyproject.toml

```diff
@@ -38,30 +38,7 @@ ignore = [
   "F403",  # undefined-local-with-import-star
   "F811",  # redefined-while-unused
 ]
-exclude = [
-  # Exclude a variety of commonly ignored directories.
-  ".bzr",
-  ".direnv",
-  ".eggs",
-  ".git",
-  ".git-rewrite",
-  ".hg",
-  ".mypy_cache",
-  ".nox",
-  ".pants.d",
-  ".pytype",
-  ".ruff_cache",
-  ".svn",
-  ".tox",
-  ".venv",
-  "__pypackages__",
-  "_build",
-  "buck-out",
-  "build",
-  "dist",
-  "node_modules",
-  "venv",
-  # Files/directories in mlflow repository
```

Comment:
/ai What does this part in pyproject.toml file do? Any risks of removing it?

Reply:
@harupy Do we need to keep some of them like "build","dist" ?

Reply:
We don't. Ruff ignores them.